include/session.hh

```diff
@@ -62,7 +73,14 @@ public:
 
     template<typename... Args,
         typename = typename std::enable_if<0 != sizeof...(Args)>::type>
-    query_result execute(const statement &statement, Args... args) {
+    query_result execute(statement &statement, Args... args) {
```

Comment:
I agree, I'd like to address it in this PR - apart from what you said, there is other problem: if you have prepared statement, you cant do something like `execute(prep.get_statement(), args...)` because execute only accepts l-value reference. I made a work-around - specialization for prepared statements, but it is a bad workaround, and I'll look into making those universal reverences.